docs/faq/troubleshooting.md

```diff
@@ -207,36 +207,59 @@ docker-compose rm -f [BROKEN_SERVICE]
 docker-compose up -d [BROKEN_SERVICE]
 ```
 
-Sometimes data for a service is volume mounted from outside the container. This persistence is good when the data is 
+Sometimes data for a service is volume mounted from outside the container. This persistence is good when the data is
 healthy, but can be really bad if the data is part of the problem (e.g., broken lockfiles).
 
 ```bash
 docker-machine ssh [MACHINE_NAME]
 rm -Rf /data/[PROJECT]/[DIRECTORY_FOR_SERVICE]
 ```
 
-Remember that your service may have configuration or data in another service, you may need to perform this operation 
+Remember that your service may have configuration or data in another service, you may need to perform this operation
 against multiple containers (e.g., Solr)
 
 ## NFS Conflicts with other Environments
 
-Running the Devtools VM in conjunction with other development environments (such as Vagrant) can sometimes cause conflicts 
-with volume mounts. This is due to the fact that the Devtools VM mounts the entirety of `/Users` into the VM on OS X. 
+Running the Devtools VM in conjunction with other development environments (such as Vagrant) can sometimes cause conflicts
+with volume mounts. This is due to the fact that the Devtools VM mounts the entirety of `/Users` into the VM on OS X.
 Additional NFS mounts from other environments that target subdirectories of `/Users` will fail.
 
-The easiest workaround is to keep projects that use non-Devtools environments like Vagrant in a directory *outside* of 
-`/Users`, such as `/opt`. Alternatively, you can run a single VM at a time and manually clear out `/etc/exports` prior 
+The easiest workaround is to keep projects that use non-Devtools environments like Vagrant in a directory *outside* of
+`/Users`, such as `/opt`. Alternatively, you can run a single VM at a time and manually clear out `/etc/exports` prior
 to switching environments/projects.
 
 ## Migrating Vagrant boxes outside of /Users
 
 If you'd like to move your vagrant boxes outside of your home directory, perform the following steps:
 
-1. Choose a destination folder.  We'll be using /opt/vms for this example. Insure the destination has enough free space. 
+1. Choose a destination folder.  We'll be using /opt/vms for this example. Insure the destination has enough free space.
 Typically this is not a problem because our Macs are one single partition.
 1. Make the new directory, and ensure your userid owns it:  `sudo mkdir /opt/vms; sudo chown -R userid:userid /opt/vms`
 1. Add `export VAGRANT_HOME=/opt/vms` to ~/.bash_profile
 1. Move your vagrant folders over to /opt/vms.
 1. edit /etc/exports, updating any vagrant mount point to use the new location.  (example: /Users/userid/vagrant/projectx/cms)
 1. While a reboot isn't necessary, it'll help to make sure nothing is running and using the old mount points.
 1. cd into your new vagrant directory, and do a vagrant up.
+
+## Permission Errors during Build Container Startup
```

Comment:
This all looks really good, except that the convention of a "Build Container" and having an ssh volume mount are opt-in conventions that outrigger supports, but are not necessarily core concepts. It would be useful to generalize this slightly, and maybe add a note that the ssh volume mount common in project build containers is a good example.